optionals/compat_rhs_gref3/CfgWeapons.hpp

```diff
@@ -87,9 +87,11 @@ class CfgWeapons {
     };
 
     class rhs_weap_mg42_base: Rifle_Base_F {
-        ACE_Overheating_allowSwapBarrel = 1;
         ACE_barrelTwist = 305.0;
         ACE_barrelLength = 530.0;
+        GVAR(allowSwapBarrel) = 1;
```

Comment:
https://ace3mod.com/wiki/development/coding-guidelines.html#macro-usage

Reply:
https://github.com/acemod/ACE3/blob/master/addons/overheating/CfgWeapons.hpp#L48

Reply:
Will expand to ace_compat_rhs_gref3_allowSwapBarrel here. Different module. No bueno. Use EGVAR macro.